### DISCUSSION TITLE
Configure ldap operators access

### DISCUSSION THREAD

**Discussion by ts2050** [LiveHelperChat/livehelperchat#1918] [General]
I'm researching how to connect livehelpder chat with my ldap base, this way the operators uses the same username and password, but I'm still a little lost on how to carry out this process in the live helper chat, would I have to configure this option in the settings?

**Comment by remdex** [comment]
See how this extension works
https://github.com/LiveHelperChat/sso-boilerplate and do that same... Most likely you will have to develop something also...
